### PR TITLE
Sales manager profile

### PR DESCRIPTION
**Pull request by sergslat**
Add custom object permissions to Application Administrator and Sales Manager profiles.

### CODE REVIEW

**Review by joseriosdev — COMMENTED** [review]
Deploy success.

File: force-app/main/default/profiles/Application Administrator.profile-meta.xml [hunk 1]
```diff
@@ -1,6 +1,38 @@
...
+    </objectPermissions>
+    <objectPermissions>
+        <allowCreate>true</allowCreate>
+        <allowDelete>true</allowDelete>
+        <allowEdit>true</allowEdit>
+        <allowRead>true</allowRead>
+        <modifyAllRecords>true</modifyAllRecords>
+        <object>Request__c</object>
+        <viewAllRecords>true</viewAllRecords>
+    </objectPermissions>
```
Comment (jamerbi) [inline force-app/main/default/profiles/Application Administrator.profile-meta.xml:35]: The application should allow the user accounts registration for Application Administrator according to the requirements.
Access to other objects could be added later.

**Comment by jamerbi** [comment]
[Create Sales Manager profile or permission set](https://trello.com/c/KWcrIIB5/6-create-sales-manager-profile-or-permission-set)

**Comment by joseriosdev** [comment]
Since App Admin profile changes were reverted, I proceed to merge this branch. Feature was tested in a Scratch Org.

**Pull request merged**